boa/src/profiler.rs

```diff
@@ -31,12 +31,12 @@ impl BoaProfiler {
             .start_recording_interval_event(kind, id, thread_id)
     }
 
-    pub fn default() -> BoaProfiler {
-        let profiler = Profiler::new(Path::new("./my_trace")).unwrap();
-        BoaProfiler { profiler }
+    pub fn default() -> Self {
+        let profiler = Profiler::new(Path::new("./my_trace")).expect("must be able to crate file");
```

Comment:
```suggestion
        let profiler = Profiler::new(Path::new("./my_trace")).expect("must be able to create file");
```
I think it is a typo